src/test/ui/type-alias-enum-variants/incorrect-variant-form-through-alias-caught.rs

```diff
@@ -0,0 +1,23 @@
+// ignore-tidy-linelength
+
+// Check that creating/matching on an enum variant through an alias with
+// the wrong braced/unit form is caught as an error.
+
+#![allow(unreachable_code)]
```

Comment:
Lints from the `unused` group are `allow`ed by default in UI tests.

Reply:
Fixed in https://github.com/rust-lang/rust/pull/61682/commits/57e6869a6be56cd75e382deb313fb5f4d1bb1fda.